client/src/components/ItemPic/ItemPic.css

```diff
@@ -0,0 +1,83 @@
+.item-pic {
+  width: 45%;
+}
+
+.item-pic__slider {
+  position: relative;
+  height: 793px;
+  width: 614px;
+  overflow: hidden;
+  margin-bottom: 30px;
+}
+
+.item-pic__images {
+  display: flex;
+  transition: all ease-in-out 0.5s
+}
+
+.item-pic__img {
+  /* this is a shorthand of flex-grow flex-shrink and flex-bases */
+  flex: 0 0 100%;
+}
+
+.item-pic__navigation {
+  position: absolute;
+  right: 10px;
+  bottom: 10px;
+}
+
+.item-pic__nav-button {
+  border: none;
+  margin-left: 1px;
+  background-color: white;
+  display: inline-block;
+  width: 36px;
+  height: 38px;
+  cursor: pointer;
+  font-size: 22px;
+  font-weight: 600;
+  text-align: center;
+  line-height: 38px;
+  box-shadow: 1px 1px 1px rgba(0,0,0,.15);
+}
+
+.item-pic__thumbnails {
+  display: flex;
+  justify-content: space-between;
+  width: 614px;
+  list-style: none;
+  padding: 0;
+}
+
+.item-pic__thumbnail {
+  position: relative;
+  width: 134px;
+  list-style: none;
+  cursor: pointer;
+  opacity: 0.5;
+  transition: all ease-in-out 0.5s;
+}
+
+.item-pic__thumbnail:hover {
+  opacity: 1;
+  transition: all ease-in-out 0.5s;
+}
+
+.item-pic__overlay {
+  position: absolute;
+  width: 100%;
+  height: 100%;
+  transition: all ease-in-out 0.5s;
+}
+.item-pic__overlay:hover, .item-pic__active .item-pic__overlay {
+  background-color: rgba(0,0,0,.08);
+  transition: all ease-in-out 0.5s;
+}
+
+.item-pic__thumbnail-img {
+  width: 100%;
+}
+
+.item-pic__active {
+  opacity: 1;
+}
```

Comment:
@ashatat add empty line at the end of the file.

Reply:
@FarahZaqout  I did

I'll wait today for @amusameh to review it, then I'll merge it. No more waiting